README.md

```diff
@@ -132,35 +132,52 @@ There are 2 APIs to set a Kusto schema:
 
 ## Changelog
 
+### 3.2.9
+
+- feat: update language service to support python code strings
```

Comment:
Minor: In versioning, a new feature requires leads to increase in the minor version (3.3.0).